rllib/models/torch/torch_action_dist.py

```diff
@@ -305,3 +309,95 @@ def sample(self):
     @override(ActionDistribution)
     def required_model_output_shape(action_space, model_config):
         return np.prod(action_space.shape)
+
+
+class TorchMultiActionDistribution(TorchDistributionWrapper):
+    """Action distribution that operates for list of (flattened) actions.
+
+    Args:
+        inputs (Tensor list): A list of tensors from which to compute samples.
+    """
+
+    def __init__(self,
+                 inputs,
+                 model,
+                 *,
+                 child_distributions,
+                 input_lens,
+                 action_space,
+                 action_space_struct=None):
```

Comment:
If the struct can always be computed from the action space, why pass both?

Reply:
Removed.